### PR TITLE
[DOC] Document how tests are being invoked

### PR DESCRIPTION
**Pull request by nonifier**
This aims at helping debugging and ease development experience.

### CODE REVIEW

**Review by benoit-bst — APPROVED** [review]

File: source/tests/readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Requirements
+ * You have already setup your environment and built Navitia following the [installation documentation](https://github.com/canaltp/navitia/blob/dev/install.rst).
+
+## How to run the tests
+
+From the your build directory, run the command:
+
+```sh
+# make test
```
Comment (mbounabi) [inline source/tests/readme.md]: why the `#` ?

**Review by mbounabi — APPROVED** [review]

File: source/tests/readme.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+`ctest` has some interesting options:
+ * `-V` to enable verbose output
+ * `-N` to disable the actual tests execution
+ * `-R <regex>` to only run tests matching the regex
+
+To get the actual command line used for `jormungandr_serpy` test for instance, you can run:
+
+```sh
+# ctest -V -N -R serpy$
```
Comment (pbench) [inline source/tests/readme.md]: This helps a lot ! :1st_place_medal:

**Review by pbench — APPROVED** [review]

File: source/tests/readme.md (hunk 1 above)
Comment (nonifier) [reply source/tests/readme.md]: you're right it should be a `$`. 
I'always confused about the `$` for a classic command and a `#` for a root command.

File: source/tests/readme.md (hunk 2 above)
Comment (nonifier) [reply source/tests/readme.md]: :pray:

**Review by kadhikari — APPROVED** [review]
:+1:

**Pull request merged**